examples/configuration/main.go

```diff
@@ -4,21 +4,23 @@ import (
 	"context"
 	"fmt"
 	"strconv"
+	"sync"
 	"time"
 
 	dapr "github.com/dapr/go-sdk/client"
 	"github.com/go-redis/redis/v8"
 	"google.golang.org/grpc/metadata"
 )
 
-func init() {
+func addItems(wg *sync.WaitGroup) {
 	opts := &redis.Options{
 		Addr: "127.0.0.1:6379",
 	}
 	client := redis.NewClient(opts)
 	// set config value
 	client.Set(context.Background(), "mykey", "myConfigValue", -1)
 	ticker := time.NewTicker(time.Second)
+	wg.Add(3*5)
```

Comment:
hardcode?

Reply:
Nope 😅 Since the loop runs 5 times and for every run there is 3 entry its 5*3 to make sure all entries that's send get recieved